### PR TITLE
Fix out of date npms

### PR DESCRIPTION
**Pull request by camelmasa**
I updated deepmerge and libp2p-crypto npms for dependencies out of date.
Also, I added `.aegir.js` file for fixing deepmerge's issue. https://github.com/KyleAMathews/deepmerge/issues/87

We can choice to wait until fix the issue. Thought ?

### CODE REVIEW

File: .aegir.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// This config is fixing deepmerge's issue https://github.com/KyleAMathews/deepmerge/issues/87.
+// We can remove it after fix webpack's issue https://github.com/webpack/webpack/issues/6584.
+
+const path = require('path')
+const config = {
+  webpack: {
+    resolve: {
+      alias: {
+        deepmerge$: path.resolve(__dirname, 'node_modules/deepmerge/dist/umd.js')
```
Comment (dryajov) [inline .aegir.js]: Do we really need `deepmerge`, can we replace it with lodash merge or defaultsDeep?

Comment (camelmasa) [reply .aegir.js]: Thanks for comment. Maybe we can replace `deepmerge` (Or, `Object.assign()` ?) . I'll investigate that.

Comment (richardschneider) [reply .aegir.js]: `Object.assign` does NOT do a deep merge.

Comment (richardschneider) [reply .aegir.js]: LGTM

**Comment by camelmasa** [comment]
I rebased commits. If it looks good, Please merge it 😃

**Pull request merged**